libs/net/dal/Services/ContentService.cs

```diff
@@ -299,8 +300,10 @@ public async Task<IPaged<API.Areas.Services.Models.Content.ContentModel>> FindWi
         if (!filter.IsHidden.HasValue)
             filterQueries.Add(s => s.Term(t => t.IsHidden, filter.IsHidden));
 
-        if (filter.HasTopic == true)
+        if (filter.HasTopic == true) {
```

Comment:
Your linter isn't following standard C# code layout

Reply:
this file wasnt meant to be part of this PR.  Attempting to undo